pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1066,16 +1066,41 @@ struct ArgSortOpConversion : public OpConversionPattern<tile::ArgSortOp> {
       // unsorted region, by setting the insertion point after its block.
       rewriter.setInsertionPointAfter(minLoop);
 
-      // Swap the sort position with the minimum position.
       auto finalMinPos =
           rewriter.create<LoadOp>(loc, minIdxVar, zeroIndex).getResult();
       ops[axis] = finalMinPos;
       auto finalMinIdx = rewriter.create<LoadOp>(loc, result, ops).getResult();
-      ops[axis] = sortIV;
-      auto origSortIdx = rewriter.create<LoadOp>(loc, result, ops).getResult();
-      rewriter.create<StoreOp>(loc, finalMinIdx, result, ops);
-      ops[axis] = finalMinPos;
-      rewriter.create<StoreOp>(loc, origSortIdx, result, ops);
+
+      switch (op.stability()) {
+      case tile::SortStability::stable: {
+        // Move every elements between [sortIV,finalMinPos) a step forward
```

Comment:
Please add reference of the stable selection sort algorithm, i.e. wikipedia or something else.

Reply:
OK but I think we should have a discussion about wether to do it this way because https://en.wikipedia.org/wiki/Selection_sort says:
> Selection sort can be implemented as a stable sort if, rather than swapping in step 2, the minimum value is inserted into the first position and the intervening values shifted up. However, this modification either requires a data structure that supports efficient insertions or deletions, such as a linked list, or it leads to performing Θ(n2) writes.

Reply:
Yes, I think for now it is good. Please add this section about performance in the PR description as well. We should add a nLogn sort algorithm in the future.

Reply:
Obviously `tile_to_pxa` conversion doesn't use linked lists so the insertion and shift-up is relatively slow.

Reply:
> Yes, I think for now it is good. Please add this section about performance in the PR description as well. We should add a nLogn sort algorithm in the future.

Sure. Will keep an eye on this when we need better performance.